auth-api/migrations/versions/975fe0b2c859_add_mhr_product_code.py

```diff
@@ -0,0 +1,30 @@
+"""Add MHR product code.
+
+Revision ID: 975fe0b2c859
+Revises: 211deb552319
+Create Date: 2022-06-02 15:37:13.873620
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '975fe0b2c859'
+down_revision = '211deb552319'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("INSERT INTO public.product_codes "
+             "(code, description,\"default\", type_code, hidden, need_review, premium_only, url) "
+             "VALUES "
+             "('MHR', 'Mobile Home Registry', true, 'INTERNAL', false, false, true, "
```

Comment:
Not sure if this title is right.. go ahead and change it if needed.

Reply:
it's "Manufactured Home Registry"